test/testdata/deployednettemplates/recipes/mainnet-model/network-tpl.json

```diff
@@ -28,7 +28,7 @@
         "participatingNodes": 0,
         "nonParticipatingNodes": 0
       },
-      "region": "us-west-1"
+      "region": "us-east-2"
```

Comment:
We had 2 reasons for using us-west-2 here:
1. It was easier to spot the generated instances, since us-west-1 wasn't used other the by this recipe.
2. There is a instance limit on each zone, and we might have reached our limit on us-east-2. ( especially when we run multiple networks concurrently ).

I can see how having the telemetry server nearby would be a good thing. I'm fine with relocating this once you verify that the latter would not break our runs.

Reply:
Our telemetry server automation sets up a self contained telemetry server right? Maybe updating that to support other regions might be better so that we can continue not worrying about instance limits in the future

Reply:
No, I think that the performance testing environment is using a pre-deployed telemetry server on us-east-1. I think that it's ok to auto-deploy a new telemetry server instance dynamically, but this means that we also need to preserve the collected data once we're done with this run.

Reply:
I made the deploy_ephemeral_telemetry.sh script take an argument for what region to run in. I think it's important to run the telemetry server in the same region as the cluster. In my experimentation in the last couple weeks it was easiest to get things running in us-east-2 but I'm not super attached to that region.